docs/2-user-guides/04-mailbox/01-corporate.md

```diff
@@ -0,0 +1,90 @@
+Ponadto mogą one być zintegrowane z workflow systemu, co umożliwia automatyzację wielu procesów biznesowych.
+
+## Konfiguracja 
+
+By móc skorzystać z tej funkcjonalności konieczne jest skonfigurowanie w panelu administracyjnym ```serwera poczty``` oraz ```skanera poczty```. Natomiast by móc wysyłać wiadomości przez system YetiForce CRM należy dodatkowo ustawić sekcję ```autologowanie```. Poniżej znajduje się opis jak można to zrobić.
+
+### Serwer poczty
+
+W celu rozpoczęcia korzystania ze skrzynki korporacyjnej, należy skonfigurować w panelu konfiguracyjnym systemu serwer pocztowy.
+Żeby to zrobić, wybierz w menu po lewej stronie ```Narzędzia pocztowe``` a następnie ```Serwery poczty```.
+W tym miejscu możemy dodać nowe konfiguracje za pomocą przycisku ```Dodaj rekord``` lub edytować aktualnie istniejące.
+
+![corporate-1](corporate-1.jpg)
+
+Po naciśnięciu przycisku ```Dodaj rekord```, będziemy mieć możliwość w pełni skonfigurować sposób podłączenia się pod usługę mailową.
+Konfiguracja imap umożliwi prawidłowe odbieranie wiadomości pocztowych a natomiast SMTP na wysyłanie wiadomości przez YetiForce z korporacyjnej skrzynki mailowej.
+
+![corporate-2](corporate-2.jpg)
+
+### Skaner poczty
+
+Kolejnym krokiem jest skonfigurowanie skanera poczty. Służy on do pobierania wiadomości mailowych z serwera pocztowego a następnie umożliwia, by w oparciu o nie wykonać odpowiednie operacje w systemie YetiForce.
+W celu jego konfiguracji z lewego menu należy wybrać ```Narzędzia pocztowe``` a następnie ```Poczta - skaner```. 
+
+![corporate-3](corporate-3.jpg)
+
+
+Przycisk ```Dodaj rekord``` umożliwi stworzenie nowego skanera poczty.
+
+![corporate-4](corporate-4.jpg)
+
+#### Pole formularza tworzenia skanera pocztowego
+- Serwer - serwer pocztowy utworzony w ```Narzędzia pocztowe``` > ```Serwery poczty```
+- Login - login służący do zalogowania się do poczty (opcjonalne przy uwierzytelnianiu OAuth)
+- Hasło - hasło umożliwiające zalogowanie się do poczty (opcjonalne przy uwierzytelnianiu OAuth)
+- Akcje skanera poczty
+    - Stwórz e-maila - wybranie tej opcji powoduje pobieranie wiadomości email z serwera pocztowego
+    - Powiąż wiadomość e-mail - pozwala połączyć wiadomość z użytkownikiem lub innym zasobem systemu na podstawie kryteriów określonych w ```Pole wyszukiwania```
+    - Otwórz zgłoszenie - ustawia odpowiedni status dla zgłoszenia powiązanego z pobraną wiadomością mailową
+    - Utwórz zgłoszenie - tworzy nowe zgłoszenie w systemie
+- Pola wyszukiwania - określa kryteria powiązania wiadomości z elementem systemu
+- Użytkownik - określa użytkownika do którego będzie przypisane utworzony elelement systemu
+
+### Wysyłanie wiadomości pocztowych
+
+Jeżeli chcemy by system YetiForce CRM nie tylko odbierał pocztę, ale także umożliwiał nam wysyłanie wiadomości pocztowych, należy w panelu administrcyjnym wybrać  ```Narzędzia pocztowe ``` a nastepnie zakładkę ```Poczta - autologowanie```.
```

Comment:
dodać <i> na początku, że ma to zastosowanie tylko w zakresie wewnętrznego klienta pocztowego (roundcube)